src/test/kotlin/no/nav/familie/ef/søknad/api/SøknadBarnetilsynControllerTest.kt

```diff
@@ -73,12 +74,12 @@ internal class SøknadBarnetilsynControllerTest : OppslagSpringRunnerTest() {
         every { featureToggleService.isEnabled(any()) } returns true
 
         val response = restTemplate.exchange<Kvittering>(
-            localhost("/api/soknad/barnetilsyn/"),
+            localhost("/api/soknad/barnetilsyn"),
```

Comment:
Feilet denne med slash? 🤔

Reply:
Eller er det for visuell opprydding? :)

Reply:
De har fjernet støtte for default match av trailing slash, sjekket frontend og virket ikke som att den bruker det